packages/shared/src/lib/featureManagement.ts

```diff
@@ -61,4 +61,17 @@ export const featureFeedAdTemplate = new Feature('feed_ad_template', {
   },
 } as Record<FeedSettingsKeys, FeedAdTemplate>);
 
+export const featureValidLanguages = new Feature('validLanguages', {
```

Comment:
This is separate flag from remoteConfig? Just want to make sure we don't leak those.

Reply:
Yeah we moving to a new one, which will be available on prod and api-config envs.

Reply:
Separate. We're also exposing this feature on API/ Kvasir. PR coming soon.

Reply:
cool cool